tests/validate/ps_aie_test/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022 Xilinx, Inc. All rights reserved.
+#
+set(TESTNAME "ps_aie.exe")
+
+set(xrt_core_LIBRARY xrt_core)
+set(xrt_coreutil_LIBRARY xrt_coreutil)
+
+include_directories(
+  ../../../src/runtime_src/core/include src/
+  ../common/includes/cmdparser
+  ../common/includes/logger
+  )
+
+add_executable(${TESTNAME}
+  ../common/includes/cmdparser/cmdlineparser.cpp
+  ../common/includes/logger/logger.cpp src/host.cpp
+  )
+
+target_link_libraries(${TESTNAME} PRIVATE ${xrt_coreutil_LIBRARY} ${Boost_FILESYSTEM_LIBRARY} ${Boost_SYSTEM_LIBRARY})
+
+if (NOT WIN32)
+  target_link_libraries(${TESTNAME} PRIVATE ${uuid_LIBRARY}  pthread ${xrt_coreutil_LIBRARY} ${xrt_core_LIBRARY})
+endif(NOT WIN32)
+
+if (NOT DEFINED XRT_VALIDATE_DIR)
+  set(XRT_VALIDATE_DIR "${CMAKE_CURRENT_BINARY_DIR}/")
+endif()
+
+install(TARGETS ${TESTNAME}
+  RUNTIME DESTINATION ${XRT_VALIDATE_DIR}
+  ) 
+
+if (XRT_STATIC_BUILD)
+  set(TESTNAME_STATIC "ps_aie_static.exe")
+  add_executable(${TESTNAME_STATIC}
+    ../common/includes/cmdparser/cmdlineparser.cpp
+    ../common/includes/logger/logger.cpp src/host.cpp
+    )
+  # Specify path directory with boost static libraries. The boost link dir
+  # should be same as the one used to build XRT. In most case this is the
+  # default /usr/lib, but if XRT was built with -with-static-boost, then
+  # the libraries cannot be located without specified the link path.
+  target_link_options(${TESTNAME_STATIC} PRIVATE "-static" "-L${Boost_LIBRARY_DIRS}")
+  target_link_libraries(${TESTNAME_STATIC}
+    PRIVATE
+    # force linking of whole archive to enable static globals
+    -Wl,--whole-archive xilinxopencl_static xrt++_static xrt_core_static xrt_coreutil_static -Wl,--no-whole-archive
+    -Wl,--whole-archive rt pthread -Wl,--no-whole-archive
+    )
+  target_link_libraries(${TESTNAME_STATIC}
+    PRIVATE
+    xrt++_static
+    xrt_core_static
+    xrt_coreutil_static
+    )
+  set_target_properties(${TESTNAME_STATIC} PROPERTIES INSTALL_RPATH "")
+  install(TARGETS ${TESTNAME_STATIC} RUNTIME DESTINATION ${XRT_VALIDATE_DIR})
+endif()
+  
```

Comment:
I don't think we should provide these as static exe.   
Let them be dynamic so as not to link exe with specific xrt_core.

If you insist on static, then at least remove opencl and xrt++, which are certainly not needed by your application